evm/src/arithmetic/modular.rs

```diff
@@ -191,8 +190,8 @@ fn bigint_to_columns<const N: usize>(num: &BigInt) -> [i64; N] {
 /// NB: `operation` can set the higher order elements in its result to
 /// zero if they are not used.
 fn generate_modular_op<F: RichField>(
-    lv: &mut [F; NUM_ARITH_COLUMNS],
-    nv: &mut [F; NUM_ARITH_COLUMNS],
+    lv: &mut [F],
+    nv: &mut [F],
     filter: usize,
```

Comment:
nit: Seems like some `generate_*` functions use `filter: usize` and some use `op: usize` instead? Is there any difference between the two? If not, maybe we could uniformize.

Reply:
Uniformised!